Extension/src/LanguageServer/configurations.ts

```diff
@@ -152,6 +152,7 @@ export class CppProperties {
     private lastCustomBrowseConfiguration: PersistentFolderState<WorkspaceBrowseConfiguration | undefined> | undefined;
     private lastCustomBrowseConfigurationProviderId: PersistentFolderState<string | undefined> | undefined;
     private lastCustomBrowseConfigurationProviderVersion: PersistentFolderState<Version> | undefined;
+    private isWin32 = os.platform() === "win32";
```

Comment:
Does this save the value or does it always have to evaluate the comparison whenever `this.isWin32` is called?

Reply:
I think it saves the value -- but why is there no type? Can you add a type "boolean"? I would've expected a linter error at least.